trivy/main.go

```diff
@@ -166,20 +166,23 @@ func (m *Scan) container(extraArgs []string) *dagger.Container {
 		WithExec(args)
 }
 
-// TODO: enabled report format enum once it's fixed
-// type ReportFormat string
-//
-// const (
-// 	Table      ReportFormat = "table"
-// 	JSON       ReportFormat = "json"
-// 	Template   ReportFormat = "template"
-// 	SARIF      ReportFormat = "sarif"
-// 	CycloneDX  ReportFormat = "cyclonedx"
-// 	SPDX       ReportFormat = "spdx"
-// 	SPDXJSON   ReportFormat = "spdx_json"
-// 	GitHub     ReportFormat = "github"
-// 	CosignVuln ReportFormat = "cosign_vuln"
-// )
+type ReportFormat string
+
+const (
+	ReportFormatTable      ReportFormat = "table"
+	ReportFormatJSON       ReportFormat = "json"
+	ReportFormatTemplate   ReportFormat = "template"
+	ReportFormatSARIF      ReportFormat = "sarif"
+	ReportFormatCycloneDX  ReportFormat = "cyclonedx"
+	ReportFormatSPDX       ReportFormat = "spdx"
+	ReportFormatSPDXJSON   ReportFormat = "spdx_json"
```

Comment:
Note - this is `spdx_json`. Your calling code refers to `spdx-json`.

This is where the error comes from, since `spdx-json` is not a valid enum identifier in the graphql schema.

Reply:
Oh, good catch, thanks!